cmd/ltops/db.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+
+	"github.com/mattermost/mattermost-load-test/ltops"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var dbCommand = &cobra.Command{
+	Use:     "db",
+	Short:   "Launches mysql connected to the cluster database",
+	PreRunE: showHelpIfNoFlags,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		clusterName, _ := cmd.Flags().GetString("cluster")
+
+		workingDir, err := defaultWorkingDirectory()
+		if err != nil {
+			return err
+		}
+
+		cluster, err := LoadCluster(filepath.Join(workingDir, clusterName))
+		if err != nil {
+			return errors.Wrap(err, "couldn't load cluster")
+		}
+
+		settings, err := cluster.DBSettings()
+		if err != nil {
```

Comment:
I'm assuming this just errors out for k8s cluster? Might be helpful to add to the help text that only terraform is supported right now

Reply:
I've exposed the `DBSettings` within the k8s cluster, but I'm uncertain if it actually works without modification: is something special needed to expose the database outside the k8s cluster?